src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -15,7 +15,7 @@ public class Conflicts : TransactionAttribute
 
         public override TransactionAttributeType Type => TransactionAttributeType.Conflicts;
 
-        public override bool AllowMultiple => true;
+        public override int AllowedCount => 4;
```

Comment:
We have 16-signers here anyway, I doubt this needs to be changed.

Reply:
We need to limit both I think, you don't think so?

Reply:
I think that we can set a low value here, and in a future if we need it, we can increase it without fork anything

Reply:
Frankly, 16 seems to be low enough for me already.